include/geode/io/mesh/detail/vtu_input_impl.hpp

```diff
@@ -113,64 +52,67 @@ namespace geode
                     if( this->match(
                             data.attribute( "Name" ).value(), "offsets" ) )
                     {
+                        OPENGEODE_EXCEPTION(
+                            this->match(
+                                data.attribute( "type" ).value(), "Int64" ),
+                            "[VTUInputImpl::read_cells] Wrong offset type" );
```

Comment:
Maybe the error message should precise which offset type we are able to manage, the help the user to know what to do?